support/hitrace-bencher/runs.json

```diff
@@ -0,0 +1,30 @@
+[
+	{
+		"args": {
+			"bencher": true,
+			"url": "https://www.google.com",
+			"tries": 5
+		},
+		"filters": [
+			{
+				"name": "Load",
```

Comment:
Does hitrace-bencher support / ignore comments in json? If yes, it would probably be quite helpful to add some commments which explain how we construct the benchmark name given the inputs.